README.md

```diff
@@ -401,9 +409,10 @@ The npm package for this project uses a semver-parsable X.0.Z version number for
 Non-release versions of this project (for example on github.com/RuntimeTools/appmetrics) will use semver-parsable X.0.Z-dev.B version numbers, where X.0.Z is the last release with Z incremented and B is an integer. For further information on the development process go to the  [appmetrics wiki][3]: [Developing](https://github.com/RuntimeTools/appmetrics/wiki/Developing).
 
 ## Version
-1.0.6
+1.0.7
 
 ## Release History
+`1.0.7` - Support for installing with a proxy.  
```

Comment:
We should probably mention `strong-oracle` support was added in `1.0.7` as well.